packages/types/src/index.ts

```diff
@@ -67,13 +67,22 @@ export interface MrkdwnElement {
   verbatim?: boolean;
 }
 
-export interface Option {
-  text: PlainTextElement | MrkdwnElement;
+export interface MrkdwnOption {
+  text: MrkdwnElement;
+  value?: string;
+  url?: string;
+  description?: PlainTextElement;
+}
+
+export interface PlainTextOption {
+  text: PlainTextElement;
   value?: string;
```

Comment:
This is a note for other reviewers. [The API document](https://api.slack.com/reference/block-kit/composition-objects#option) says `value` is not optional. However, it actually can be absent. In this case, the sent value of the block in block_actions / view_submission payloads is null. Thus, we should keep this as optional.